### PR TITLE
Listing Datasources

### PR DESCRIPTION
**Pull request by Nepomuceno**
### What does this PR do?

Datasources listing currently just listing the datasources.

image

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

**Review by sebastus — APPROVED** [review]
lgtm

File: client/src/api/Models.ts [hunk 1]
```diff
@@ -41,6 +41,12 @@ export const DEFAULT_FFT_PARAMETERS: FFTParams = {
   magnitude_max: 20,
 };
 
+export enum ClientType {
+  API = 'api',
+  LOCAL = 'local',
+  BLOB = 'azure_blob',
+}
+
 export const CLIENT_TYPE_API = 'api';
```
Comment (Jade-Codes) [inline client/src/api/Models.ts:50]: why keep these?

File: client/src/pages/admin/admin.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ export const Admin = () => {
             </li>
           </ul>
         </div>
-        <div className="w-2/5 p-2">
+        <div className="w-3/4 p-2">
```
Comment (Jade-Codes) [inline client/src/pages/admin/admin.tsx:50]: disagree

File: client/src/pages/admin/pages/__tests__/data-sources.test.tsx [hunk 3]
```diff
@@ -3,10 +3,45 @@ import { render, screen } from '@testing-library/react';
...
+          sasToken: 'test',
+        },
+      ]
+    );
+    render(<DataSources></DataSources>, {
+      wrapper: wrapper,
+    });
     expect(await screen.findByRole('heading', { name: 'Data Sources' })).toBeInTheDocument();
+    expect(screen.getByText('Test Blob')).toBeInTheDocument();
+    expect(screen.getByText('Test API')).toBeInTheDocument();
```
Comment (Jade-Codes) [inline client/src/pages/admin/pages/__tests__/data-sources.test.tsx:45]: And the rest

File: client/src/api/Models.ts (hunk 1 above)
Comment (Nepomuceno) [reply client/src/api/Models.ts:50]: Will remove them but thought about doing that in another pr

File: client/src/pages/admin/admin.tsx (hunk 2 above)
Comment (Nepomuceno) [reply client/src/pages/admin/admin.tsx:50]: You are wrong 🤣

File: client/src/pages/admin/pages/__tests__/data-sources.test.tsx (hunk 3 above)
Comment (Nepomuceno) [reply client/src/pages/admin/pages/__tests__/data-sources.test.tsx:45]: Isn't seeing this enough?

File: client/src/api/Models.ts (hunk 1 above)
Comment (Jade-Codes) [reply client/src/api/Models.ts:50]: It's just a temporary *cough* permament *cough* solution

**Comment by Jade-Codes** [comment]
It is not s**t

**Review by Jade-Codes — APPROVED** [review]

**Pull request merged**